ci/gpu/build.sh

```diff
@@ -68,6 +68,7 @@ conda install -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaul
       dask-cuda=${MINOR_VERSION} \
       scikit-learn>=0.21 \
       nccl>=2.5 \
```

Comment:
You removed nccl from the environment files. Should it also be removed here?

Reply:
nccl is still needed in the environment. I removed nccl as it was redundant. We already have a nccl>=2.5 dependency